network/src/commonMain/kotlin/com/wire/kalium/network/api/base/authenticated/conversation/ConversationResponse.kt

```diff
@@ -73,8 +74,17 @@ data class ConversationResponse(
     @SerialName("cipher_suite")
     val mlsCipherSuiteTag: Int?,
 
-    @SerialName("access") val access: Set<ConversationAccessDTO>,
-    @SerialName("access_role_v2") val accessRole: Set<ConversationAccessRoleDTO> = ConversationAccessRoleDTO.DEFAULT_VALUE_WHEN_NULL,
+    @SerialName("access")
+    val access: Set<ConversationAccessDTO>,
+
+    /**
+     * **Deprecation info**: Since API v3 `access_role_v2` is deprecated and replaced by `access_role`,
+     * when the minimum supported version is API v3 we can remove `access_role_v2`.
+     *
+     * Further info: https://wearezeta.atlassian.net/wiki/spaces/ENGINEERIN/pages/672006169/API+changes+v2+v3
+     */
+    @SerialName("access_role_v2") @JsonNames("access_role")
```

Comment:
as discussed yesterday we need to wait on a confirmation form the BE team on changing the `access_role` to avoid clashing with the legacy value `access_role`

Reply:
In this case I can make a V3 specific `ConversationResponse`

Reply:
Created a `ConversationResponseV3` so there's no need for `JsonNames`.